tutorials/metrics/plot_metrics.py

```diff
@@ -1,117 +1,247 @@
-plt.ylabel('Calinski Harabasz score')
+# Compare scores
+# --------------
+# We can compare the different scores on a barplot using
+# :func:`matplotlib.pyplot.bar`. However, each score spans a different scale,
+# often several order of magnitude different from the others. First, we will
+# normalize each score to unit norm. Except for Davies-Bouldin score, the
+# general rule is the higher the better. Thus, the Davies-Bouldin scores will
+# be inverted.
+
+# normalize scores using sklearn
+from sklearn.preprocessing import normalize
+scores = {score: normalize(value[:, np.newaxis], axis=0).ravel()
+          for score, value in scores.items()}
+# invert davies-bouldin scores
+scores["Davies-Bouldin"] = 1 - scores["Davies-Bouldin"]
+
+# set width of a bar and define colors
+barWidth = 0.2
+colors = ["#4878D0", "#EE854A", "#6ACC64", "#D65F5F"]
+
+# create figure
+plt.figure(figsize=(10, 8))
+# create the position of the bars on the X-axis
+x = [[elt + k * barWidth for elt in np.arange(len(cluster_numbers))]
+     for k in range(len(scores))]
+# create plots
+for k, (score, values) in enumerate(scores.items()):
+    plt.bar(
+        x=x[k],
+        height=values,
+        width=barWidth,
+        edgecolor="grey",
+        color=colors[k],
+        label=score,
+    )
+# add labels and legend
+plt.xlabel("Number of clusters")
+plt.ylabel("Score normalize to unit norm")
+plt.xticks(
+    [pos + 1.5 * barWidth for pos in range(len(cluster_numbers))],
+    [str(k) for k in cluster_numbers],
+)
+plt.legend()
 plt.show()
 
 #%%
-# Dunn score
-
-plt.figure()
-plt.scatter(cluster_numbers, dunn_scores)
-plt.xlabel('n_clusters')
-plt.ylabel('Dunn score')
-plt.show()
+# Conclusion
+# ----------
+# There is no global consensus on which ``n_cluster`` value to
+# choose. The Silhoutette score seems to be in favor of
+# ``n_clusters=3``, Calinski-Harabasz score ``n_clusters=4``,
+# Dunn score ``n_clusters=5`` and Davies-Bouldin score ``n_clusters=2 or 5``.
+# Each of these scores provides similar but different information about the
+# quality of the clustering.
+#
+# Note that for microstate analysis the choice of the number of
+# :term:`cluster centers` is also a trade off between the quality of the
+# clustering, interpretations that can be made of it, and the variance
+# expressed by the current clustering.
+# There isn't an ideal solution and it is up to the person who conducts the
+# analysis to evaluate the most judicious choice, by exploring for example
+# several clustering solutions.
 
 #%%
-# Davies Bouldin
+# Microstates fitted for n=5
+# --------------------------
+# In this case, ``n_clusters=5`` seems like a reasonable choice. This number of
+# clusters yields the following maps:
 
-plt.figure()
-plt.scatter(cluster_numbers, davies_bouldin_scores)
-plt.xlabel('n_clusters')
-plt.ylabel('Davies Bouldin score')
-plt.show()
+ModK = ModKMeans(n_clusters=5, random_state=42)
+ModK.fit(gfp_peaks, n_jobs=2, verbose="WARNING")
+ModK.plot()
```

Comment:
I tried to fit only the GFP peaks of the first 3 minutes. Could you check if you agree with what was written, and could you reorder/rename the microstates in the last plots if it looks correct? @vferat 

Also, if it's correct, it might be good to try to lower the duration of the recording to speed-up drastically the build. Maybe down to 2 minutes, or 1 minutes, as long as it yields relevant results.

Reply:
Maps are ok

I like the current results for 3min, and didn't manage to find similar one with other parameters. I think it's worth keeping it because it is an ideal scenario of where metrics are interesting.

Reply:
Can you also give me the standard order? I'd like to reorder/rename them.

Reply:
It should already be done ( pre-visualisation is outdated)